postgresql/README.md

```diff
@@ -12,7 +12,7 @@ hab start core/postgresql
 ```
 or
 ```
-docker run starkandwayne/postgresql
+docker run chefdemo/postgresql
```

Comment:
What would the origin be here after merge?

Reply:
ah, I just realized that something (bldr?) is publishing official versions of this plan to https://hub.docker.com/r/habitat/postgresql/ so that would make more sense here

Reply:
fixed!